packages/service/package.json

```diff
@@ -9,6 +9,9 @@
     "type": "git",
     "url": "https://github.com/webdriverio/visual-testing.git"
   },
+  "bin": {
+    "visual-service": "./dist/cli.js"
```

Comment:
Thinking out loud here: should make this an extension to the `wdio` CLI? e.g. `wdio visual`?

Reply:
Hi @christian-bromann 

What do you exactly mean by this?